src/com/moclojer/specs/moclojer.clj

```diff
@@ -55,6 +60,19 @@
                       [(name k) (str v)]))
                (:headers response))}))
 
+(defn generic-reitit-handler [response webhook-config]
+  (fn [{path :parameters}]
+    (log/log :info :request path)
+    (let [body (build-body response path)]
+      (log/log :info :body (json/read-str body :key-fn keyword))
+      {:body  (json/read-str body :key-fn keyword)
+       :status (:status response)
+       :headers (into
+                 {}
+                 (map (fn [[k v]]
+                        [(name k) (str v)]))
+                 (:headers response))})))
```

Comment:
the handler is kind different from pedestal, 
` "hello": "{{path-params.username}}!" ->           "hello": "{{path.username}}!"
`